pykern/pkcli/ci.py

```diff
@@ -124,8 +124,14 @@ def _error(m):
     n = 0
     for f in pkio.walk_tree(p, d.include_files):
         f = p.bestrelpath(f)
-        if re.search(d.exclude_files, f):
+        if re.search(d.exclude_files, f) or (
+            github_ci_repo
+            and not re.search(
+                re.compile(github_ci_repo + r"/|tests/|README|setup\.py$"), f
+            )
+        ):
             continue
+
```

Comment:
Unnecessary whitespace. If you feel the need for whitespace, probably time to make a separate function

Reply:
fmt added that